charts/hono/values.yaml

```diff
@@ -20,6 +20,12 @@
 # Alternatively, the tag name can also be set per component image.
 # honoImagesTag:
 
+# honoContainerRegistry contains the name of the container registry that
+# the container images for Hono's components should be pulled from.
+# If not set explicitly, images are pulled from Docker Hub, which means that
+# 'index.docker.io' is used as the registry name.
+# honoContainerRegistry
```

Comment:
Missing example value.
`# honoContainerRegistry: index.docker.io`
Or at least add colon.

Reply:
yes, I will set the default value here explicitly ...